assets/facets.js

```diff
@@ -79,6 +79,13 @@ class FacetFiltersForm extends HTMLElement {
     document.getElementById('ProductGridContainer').innerHTML = new DOMParser()
       .parseFromString(html, 'text/html')
       .getElementById('ProductGridContainer').innerHTML;
+
+    document
+      .getElementById('ProductGridContainer')
+      .querySelectorAll('.grid__item')
```

Comment:
Nit: It's the same elements, but would it be more self-documenting to select elements with the `scroll-trigger` class?